lc2fen/infer_pieces.py

```diff
@@ -81,34 +81,34 @@ def __max_piece(tops):
     Returns the index of the piece with max probability.
     """
     # Set the initial maximum probability and index to the first piece in the list
-    value = tops[0][0][0]  # B(White Bishop)
+    value = tops[0][0]["B"]  # B(White Bishop)
     idx = 0
-    if tops[1][0][2] > value:  # N(White Knight)
-        value = tops[1][0][2]
+    if tops[1][0]["N"] > value:  # N(White Knight)
+        value = tops[1][0]["N"]
         idx = 1
-    if tops[2][0][3] > value:  # P(White Pawn)
-        value = tops[2][0][3]
+    if tops[2][0]["P"] > value:  # P(White Pawn)
+        value = tops[2][0]["P"]
         idx = 2
-    if tops[3][0][4] > value:  # Q(White Queen)
-        value = tops[3][0][4]
+    if tops[3][0]["Q"] > value:  # Q(White Queen)
+        value = tops[3][0]["Q"]
         idx = 3
-    if tops[4][0][5] > value:  # R(White Rook)
-        value = tops[4][0][5]
+    if tops[4][0]["R"] > value:  # R(White Rook)
+        value = tops[4][0]["R"]
         idx = 4
-    if tops[5][0][7] > value:  # b(Black Bishop)
-        value = tops[5][0][7]
+    if tops[5][0]['b'] > value:  # b(Black Bishop)
```

Comment:
For consistency could you use double quotes (") instead of single ones (') here? If you can, use the "black" python formatter.